reports/sig-security/2017-07-19.md

```diff
@@ -0,0 +1,22 @@
+# 2017-07-19
+Time: **9am PDT** (12pm EDT, 5pm BST) [see the time in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Linuxkit+Security+SIG&iso=20170719T09&p1=224)
+
+Meeting location: https://docker.zoom.us/j/779801882
+
+Announcement: Coming soon
+
+Previous meeting notes: [2017-07-05](2017-07-05.md)
+
+## Agenda
+- Introductions & Administrivia (5 min)
+- WireGuard deep dive - @zx2c4 (45 min)
+  - Differences from other VPNs
+  - Signal Protocol
```

Comment:
WireGuard is not the signal protocol. Not sure why you wrote this. If you need to write something, just write "Cryptography". In any case, I'll probably give a slightly different presentation than the one you listed here.